app/scripts/lib/rpc-method-middleware/handlers/ethereum-chain-utils.js

```diff
@@ -120,7 +120,7 @@ export function validateAddEthereumChainParams(params, end) {
 
   const firstValidRPCUrl = rpcUrls.find((rpcUrl) => isLocalhostOrHttps(rpcUrl));
   const firstValidBlockExplorerUrl =
-    blockExplorerUrls !== null && Array.isArray(blockExplorerUrls)
+    blockExplorerUrls && Array.isArray(blockExplorerUrls)
```

Comment:
```
> Array.isArray(null)
false
> Array.isArray(undefined)
false
> Array.isArray(undefined)
false
> Array.isArray('')
false
> Array.isArray([])
true
```

Does the first part of the check need to be there in the first place?

Reply:
Seems like we can scrap it!

Reply:
done here https://github.com/MetaMask/metamask-extension/pull/26938/commits/cb9d89e66c349d5eb045c23fcfb85d4ee23601d0